code/script.js

```diff
@@ -1,47 +1,103 @@
+  <select id='select'>
+    <option value = '' selected disabled> Click here to chose subject</option>
+    <option id='topicOption' value = 'My childhood'>My Childhood</option>
+    <option id='topicOption' value = 'My mother'>My Mother</option>
+    <option id='topicOption' value = 'My partner'>My Partner</option>
+  </select>
+</form>
+` 
 
-// This function will add a chat bubble in the correct place based on who the sender is
+const newTextInput = `
+<form id='partner-form'>
+  <input id='input-value' type='text' />
+    <button class='send-btn' type='submit'>
+        Send
+   </button>
+</form>`
+
+// Functions declared here
 const showMessage = (message, sender) => {
-  if (sender === 'user') {
-    chat.innerHTML += `
-      <section class="user-msg">
-        <div class="bubble user-bubble">
-          <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
-      </section>
-    `
+  if (sender === 'user') {  
+  chat.innerHTML += `
+    <section class='user-msg'>
+      <div class='bubble user-bubble'>
+        <p>${message}</p>
+      </div>
+      <img src='assets/user2.png' alt='User' />  
+    </section>
+  `
   } else if (sender === 'bot') {
-    chat.innerHTML += `
-      <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
-        </div>
-      </section>
-    `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
+   chat.innerHTML += `
+    <section class='bot-msg'>
+      <img src='assets/bot2.png' alt='Bot' />
+      <div class='bubble bot-bubble'>
+        <p>${message}</p>
+      </div>
+    </section>
+  `
+  } chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Question 1 - Start here with greeting and ask for name
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage (`Welcome to your first therapy session! What's your name?`, 'bot')
+  form.addEventListener ('submit', (event) => { 
+    event.preventDefault ()
+    handleNameInput ()
+  })
+}
+
+setTimeout (greeting, 1000)
+
+
+//Question 2 - Choose topic of session
+const handleNameInput = (event) => {
+  const userName = nameInput.value
+  showMessage (userName, 'user')
+  nameInput.value = ''
+  showMessage (`Hello ${userName}! What would you like to talk about today?`, 'bot')
+  inputWrapper.innerHTML = topicChoice; 
+  const selectedTopic = document.getElementById ('select')
+  selectedTopic.addEventListener ('change', () => {
+  showMessage (selectedTopic.value, 'user')
+  tellMeMore (selectedTopic)
+})}
+
+
+//Question 3 - Tell me more
+const tellMeMore = (selectedTopic) => {
+  if (selectedTopic.value === 'My childhood') {
+  showMessage('Could you tell me about your childhood?', 'bot')
```

Comment:
Your messages are nearly the same, they could be written once as a template with `${value}`  and reuse here 😸